docker/python-deb/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM python:2.7.15-slim-stretch
+
+# Basic linux utilities
+RUN apt-get update \
+  && apt-get install -y \
+    procps
+
+COPY requirements.txt .
+
+RUN pip install --no-cache-dir -r requirements.txt
+
+RUN groupadd --gid 4000 demisto \
```

Comment:
did you test these docker images work well in demisto ? on various OS's ?

Reply:
Only tested locally (mac). Need to discuss how we want to test them. I am open for suggestions. We can take this offline.